modules/proc_deploy_quay_add.adoc

```diff
@@ -78,7 +78,7 @@ Clair images scanning and Repository Mirroring, continue on to the next section.
 == Add Clair image scanning to {productname}
 
 Setting up and deploying Clair image scanning for your
-{productname} deployment is described in link:manage_quay.html#clair-v4[Clair Security Scanning] 
+{productname} deployment is described in link:https://access.redhat.com/documentation/en-us/red_hat_quay/3.5/html-single/manage_red_hat_quay/index#clair-v4[Clair Security Scanning] 
```

Comment:
Instead of `3.5`, use `{producty}`.
You may also wish to consider where the upstream docs are located and include the conditional on `upstream` and `downstream`.

Reply:
Yes, we definitely should use variables from modules/attributes.adoc - if we have to hardcode, then use /3/ as it will always point to the latest version.

As regards upstream vs downstream, the entire doc set needs to be reviewed to identify what needs to be conditionalized, so that can be left for a separate JIRA.